src/defines.pri

```diff
@@ -47,7 +47,13 @@ win32-msvc* {
 }
 
 # QtDBus not available on Mac
-mac: DEFINES *= DISABLE_DBUS
+mac {
+    DEFINES *= DISABLE_DBUS
+
+    # Git revision
```

Comment:
Instead of copying the same code from unix conditional, move it to conditional that is for both unix & mac.